.github/workflows/release.yml

```diff
@@ -21,6 +21,62 @@ jobs:
         echo $RELEASE_VERSION
         echo ${{ steps.vars.outputs.release_version }}
 
+  build-publish-docker-images:
+    runs-on: [ubuntu-latest]
+    strategy:
+      matrix:
+        component: [core, serving, jupyter]
+    env:
+      MAVEN_CACHE: gs://feast-templocation-kf-feast/.m2.2020-08-19.tar
+      DOCKER_BUILDKIT: '1'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: '290.0.1'
+          export_default_credentials: true
+      - run: gcloud auth configure-docker --quiet
```

Comment:
What is this for?

Reply:
If I'm understanding it correctly, it configures the credentials for the github action since we're not providing the SA key explicitly in the previous step as shown in the example [here](https://github.com/GoogleCloudPlatform/github-actions/tree/master/setup-gcloud#usage).

Reply:
@terryyylim gcloud auth configure-docker is only used when you need to authenticate docker to push to Google container registry.